packages/icons/src/Svg.js

```diff
@@ -58,6 +58,7 @@ const Svg = styled('svg')(
     flex: 'none'
   },
   space,
+  width,
```

Comment:
Should we include the propTypes for width since we're using it now?

`...width.propTypes`